docs/docs/genai/eval-monitor/insights/analyze-experiment.mdx

```diff
@@ -0,0 +1,127 @@
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+import { MessageSquare, Target, BarChart3, ExternalLink, Bot, Users, CheckCircle, TrendingUp, Database, Book, Rocket, Scale } from "lucide-react";
+
+# Analyze Experiment
+
+Automatically analyze traces in your MLflow experiments to identify operational issues, quality problems, and performance patterns. The Analyze Experiment tool uses hypothesis-driven analysis to systematically examine your GenAI application's behavior and generate comprehensive markdown reports.
+
+## Overview
+
+The Analyze Experiment command examines traces logged in an MLflow experiment to automatically discover:
+
+- **Operational Issues**: Errors, timeouts, rate limiting, authentication failures, and performance bottlenecks
+- **Quality Issues**: Overly verbose responses, inconsistent outputs, repetitive content, and inappropriate response formats
+- **Success Patterns**: What's working well, effective tool usage, and high-quality interactions
+- **Performance Metrics**: Latency distributions, success rates, and error patterns
+
+The tool generates a detailed markdown report with specific trace examples, quantitative evidence, and actionable recommendations for improvement.
+
+## Usage
+
+The Analyze Experiment functionality is available through two methods:
+
+<Tabs>
+<TabItem value="mcp" label="MCP" default>
+
+### Using MCP
+
+If you have MLflow's MCP server configured, you can simply run:
+
+```
+/analyze-experiment
+```
+
+### Prerequisites
+- MLflow MCP server (see [MCP setup guide](/genai/mcp/))
+- A coding agent with MCP support (e.g., Claude Code, Cursor, Windsurf, etc.) configured to connect to MLflow MCP server
+- MLflow experiment with logged traces
+
+</TabItem>
+<TabItem value="cli" label="CLI">
+
+### Using CLI
+
+Run the analyze experiment command directly:
+
+```bash
+mlflow ai-commands genai/analyze-experiment
+```
+
+### Prerequisites
+- A coding agent with MCP support (e.g., Claude Code, Cursor, Windsurf, etc.)
+- MLflow 3.4 or later: `pip install mlflow>=3.4`
+- Access to MLflow tracking server with logged traces
+
+</TabItem>
+</Tabs>
+
+## Analysis Workflow
+
+### 1. Setup and Authentication
+
+The tool will ask you to configure authentication:
+
+- **Databricks**: Provide workspace URL and personal access token, or use a Databricks CLI profile
```

Comment:
Each of the sections here might look a little repeated. Can some of the elements be broken up and rendered as something other than bulleted lists? Take a look at the tracing docs or the scorers docs to see examples to get inspiration from